telegram/ext/filters.py

```diff
@@ -159,6 +168,7 @@ class _Text(BaseFilter):
         name = 'Filters.text'
 
         def filter(self, message):
+            print('text_filter_filter {}'.format(repr(self)))
```

Comment:
why do you `print` here?
also nitpicking: `'text_filter_filter {:r}'.format(self)` is more idiomatic